frontend/packages/console-app/src/components/nodes/NodesPage.tsx

```diff
@@ -622,6 +642,35 @@ const getFilters = (t: TFunction): RowFilter<NodeRowItem>[] => [
       return input.selected?.some((r) => nodeRoles.includes(r));
     },
   },
+  {
+    filterGroupName: t('console-app~CPU Architecture'),
+    type: 'cpu-arch',
+    reducer: (obj) => (isCSRResource(obj) ? 's390x' : getNodeCpuArch(obj)),
+    items: [
+      { id: 'amd64', title: t('console-app~amd64') },
+      { id: 'ppc64le', title: t('console-app~ppc64le') },
+      { id: 'arm64', title: t('console-app~arm64') },
+      { id: 's390x', title: t('console-app~s390x') },
+    ],
+    filter: (input, obj) => {
+      if (!input.selected?.length) {
+        return true;
+      }
+      if (isCSRResource(obj)) {
+        return input.selected?.includes('amd64');
+      }
+      if (isCSRResource(obj)) {
+        return input.selected?.includes('ppc64le');
+      }
+      if (isCSRResource(obj)) {
+        return input.selected?.includes('arm64');
+      }
+      if (isCSRResource(obj)) {
+        return input.selected?.includes('s390x');
+      }
+      return input.selected?.includes(getNodeCpuArch(obj));
```

Comment:
This generates the list dynamically? something of this doesn't feel right. the input could be amd64/s390x in the filter, and would need to be combined into one list/set?

Reply:
ok we can do that , but once we get the z system added to the cluster we can test , based on that we can change the list.